inc/extras.php

```diff
@@ -35,8 +35,7 @@ function twentyseventeen_body_classes( $classes ) {
 	}
 
 	// Add class if no custom header or featured images
-	$header_image = get_header_image();
-	if ( '' == $header_image && ( ! has_post_thumbnail( $post->ID ) || is_home() ) ) {
+	if ( false === has_header_image() && ( ! has_post_thumbnail() || is_home() ) ) {
```

Comment:
I think you should use one of them for every check in the statement

```php
if ( false === has_header_image() && ( false ===  has_post_thumbnail()
```

OR

```php
if ( ! has_header_image() && ( ! has_post_thumbnail() 
```

Guess, now it would be more clear what I was talking about.